src/components/chart.vue

```diff
@@ -0,0 +1,49 @@
+<script>
+import scatter from '../assets/js/scatter'  
+
+export default {
+    data() {
+        return {
+            myScatter:null,
+            data:[]
+        }
+    },
+    // 先获取数据，然后再开始绘图
+    created(){
+        this.getData()
+    },
+    methods:{
+        // 文件获取的api需要单独写在一个文件里
+        getData(){
+            const request = new XMLHttpRequest();
+            request.open("get",'/json/committer.json');
```

Comment:
If this file is processed by some program, we need to add the program into repo.